python/foxglove-sdk/CONTRIBUTING.md

```diff
@@ -6,12 +6,13 @@
 
 We use [Poetry](https://python-poetry.org/) to manage dependencies.
 
-Install Poetry:
+Install Poetry and Maturin:
 
 ```sh
 brew install pipx
 pipx ensurepath
 pipx install poetry
+pipx install maturin
```

Comment:
Was this required for you? maturin should be installed as part of `poetry install`, and then we run the commands with `poetry run maturin ...`

Reply:
Might've done something wrong when I was getting things set up for the first time...I can remove it